src/convertToObject.js

```diff
@@ -1,18 +1,20 @@
-'use strict';
+"use strict";
 
-/**
- * Implement convertToObject function:
- *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
- * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
- *
- * @param {string} sourceString
- *
- * @return {object}
- */
 function convertToObject(sourceString) {
-  // write your code here
+  const styles = sourceString.split(";");
+  const stylesCSS = {};
+
+  styles
+    .filter((style) => (style !== "" ? style.trim() : false))
+    .map((style) => {
```

Comment:
- It doesn't make sense to use `map` if you don't need returned from it array
- also, you can generate an object with `reduce`